data/cptdata.py

```
@@ -0,0 +1,144 @@
+        'entigraph_pruned_subset_95': f'{bin_data_dir3}/quality_all-entigraphgpt-4-turbo-subset_95.0.bin',
+        'entigraph_pruned_subset_97': f'{bin_data_dir3}/quality_all-entigraphgpt-4-turbo-subset_97.0.bin',
+
+        'official_qa_sft':       f'{bin_data_dir}/quality_qasftgpt-4-turbo.bin',  # in fact qa continued pretraining
+        'official_qa_sft_mcq':       f'{bin_data_dir}/quality_qasft_mcqgpt-4-turbo.bin',
+        'flow_0.1':              f'{bin_data_dir}/flow_0.1.bin',  # our generated qa continued pretraining data
+
+        # corpus instruction tuning data
+        # TODO
+
+        # IL_qasft_v1
+        'IL_qasft_v1_train':       f'IL_qasft_v1_train.bin',
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)